services/BasicTypes.proto

```diff
@@ -99,6 +99,12 @@ message TokenTransferList {
     repeated NftTransfer nftTransfers = 3; // Applicable to tokens of type NON_FUNGIBLE_UNIQUE. Multiple list of NftTransfers, each of which has a sender and receiver account, including the serial number of the NFT
 }
 
+/* Representing the percentage of tokens to be transferred as part of the CustomFee */
+message Fraction {
+    uint64 numerator = 1; // numerator of the fraction
+    uint64 denominator = 2; // denominator of the fraction
+}
```

Comment:
The first letter of the word in the comment should be in uppercase to match the rest

Reply:
Done.